cli/lib/kontena_cli.rb

```diff
@@ -26,11 +26,20 @@ def self.version
   end
 
   def self.pastel
-    @pastel ||= Pastel.new(enabled: $stdout.tty?)
+    @pastel ||= Pastel.new(enabled: (ENV['OS'] != 'Windows_NT' && $stdout.tty?))
   end
 
   def self.prompt
-    @prompt ||= TTY::Prompt.new(
+    return @prompt if @prompt
+    if ENV['OS'] == 'Windows_NT'
```

Comment:
This is another: `RUBY_PLATFORM =~ /mswin/`

With the env it's easier to try out on non-windows.